tests/components/plex/test_config_flow.py

```diff
@@ -52,8 +64,8 @@ async def test_bad_credentials(hass):
         result = await hass.config_entries.flow.async_configure(result["flow_id"])
 
         assert result["type"] == "form"
-        assert result["step_id"] == "start_website_auth"
-        assert result["errors"]["base"] == "faulty_credentials"
+        assert result["step_id"] == "user"
+        assert result["errors"][CONF_TOKEN] == "faulty_credentials"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)